pubnub/dtos.py

```diff
@@ -31,17 +39,19 @@ def __init__(self, channels=None, channel_groups=None):
         self.channels = channels
         self.channel_groups = channel_groups
 
-    def get_subscribed_channels(self, channels, with_presence=False) -> list:
-        result = [ch for ch in channels if ch not in self.channels and not ch.endswith('-pnpres')]
-        if not with_presence:
-            return result
-        return result + [ch + '-pnpres' for ch in result]
-
-    def get_subscribed_channel_groups(self, channel_groups, with_presence=False) -> list:
-        result = [grp for grp in channel_groups if grp not in self.channel_groups and not grp.endswith('-pnpres')]
-        if not with_presence:
-            return result
-        return result + [grp + '-pnpres' for grp in result]
+    def get_subscribed_channels(self, channels) -> list:
```

Comment:
I'm curious, why do you need a method like this in `UnsubscribeOperation`?

Reply:
This is because of weird inconsistency in the subscription manager underneath. `UnsubscribeOperation` is a DTO and I need a list of channels that are still subscribed and list of channels that I'm leaving so instead of refactoring everything all at once I made a list explicitly saying what channels are subscribed and a list of channels I leave